packages/devextreme/js/__internal/ui/html_editor/utils/m_ai.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  HtmlEditorAIChangeStyleOption,
+  HtmlEditorAIChangeToneOption,
+  HtmlEditorAICommandName,
+  HtmlEditorAIToolbarItem,
+  HtmlEditorAITranslateOption,
+} from '@js/ui/html_editor';
+
+type DefaultOptionName =
+  | HtmlEditorAIChangeStyleOption
+  | HtmlEditorAIChangeToneOption
+  | HtmlEditorAITranslateOption;
+
+export interface CommandDefinition {
+  id: string;
+  text: string;
+  options?: DefaultOptionName[];
+}
+
+export type CommandsMap = Record<string, CommandDefinition>;
+
+export const capitalize = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
+
+const createDisplayNameMap = <T extends string>(
+  items: readonly T[],
+): Record<T, string> => items.reduce((acc, item) => {
+    acc[item] = capitalize(item);
+    return acc;
+  }, {} as Record<T, string>);
+
+export const defaultCommandNames: Record<HtmlEditorAICommandName, string> = {
+  summarize: 'Summarize',
+  proofread: 'Proofread',
+  expand: 'Expand',
+  shorten: 'Shorten',
+  changeStyle: 'Change Style',
+  changeTone: 'Change Tone',
+  translate: 'Translate',
+  askAI: 'Ask AI',
+};
+
+export const htmlEditorAIChangeStyleOptions: HtmlEditorAIChangeStyleOption[] = [
+  'formal', 'informal', 'technical', 'business',
+  'creative', 'journalistic', 'academic', 'persuasive',
+  'narrative', 'expository', 'descriptive', 'conversational',
+];
+
+export const htmlEditorAIChangeToneOptions: HtmlEditorAIChangeToneOption[] = [
+  'professional', 'casual', 'straightforward', 'confident', 'friendly',
+];
+
+export const htmlEditorAITranslateOptions: HtmlEditorAITranslateOption[] = [
+  'arabic', 'chinese', 'english', 'french', 'german', 'japanese', 'spanish',
+];
+
+const defaultCommandOptions: Record<
```

Comment:
[naming]
`defaultCommandOptions` sounds like it's a list of options
```suggestion
const commandToOptionsMap: Record<
```